package.json

```diff
@@ -123,6 +100,6 @@
   ],
   "license": "MIT",
   "engines": {
-    "node": ">=16.15.1"
+    "node": "^20.19.0 || ^22.12.0 || >=23"
```

Comment:
I assume these versions for unflagged support of `require(esm)`? I agree with this. 

Could allow older versions since they would work with esm and flagged support for `require(esm)`, but simpler story and support and user experience to have it Just Work if installs.

Reply:
Someone using pure ESM should be able to use a looser version range of Node.js, it's relatively rare that people actually install with `--engine-strict` I think?